### DISCUSSION TITLE
Date / Timestamp

### DISCUSSION THREAD

**Discussion by chprojectsupport** [cortexrd/Knack-Toolkit-Library#303] [Ideas]
Hi Guys, I'm just showing a client how to setup KTL and just showed the _ts timestamp keyword. Is there an option to have a date component to that so we can see the last date/time the grid was updated?

**Comment by cortexrd** [comment]
Carl,

Your request has been fulfilled!
This has been added in the release v0.26.1.

Cheers,
Norm

**Comment by chprojectsupport** [comment]
Whoop, whoop, thank you Norm....you are a star ⭐⭐⭐⭐⭐

Carl Holmes
Director
Mobile 07768 128128
www.chprojectsupportlimited.com

**Reply by Deanobeano2** [reply]
Norm,

By any chance did you break something while implementing this feature request as the word "false" now appears before the timestamp? Or, is there now another condition for _ts?

![image](https://github.com/cortexrd/Knack-Toolkit-Library/assets/130625963/02ebaf52-29e9-4398-b304-724790edee1f)

Dean

**Comment by cortexrd** [comment]
@Deanobeano2,

Yes this was broken indeed.
But actually, I had finished coding the fix a minute ago!

So the release has been pushed. 

Try latest code:  **v0.26.1**.

Norm
